CSharpMath/Typesetting/Typesetter.cs

```diff
@@ -1090,12 +1108,24 @@ public class Typesetter<TFont, TGlyph>
           var line = new TextLineDisplay<TFont, TGlyph>(runs, atoms) {
             Position = _currentPosition
           };
-          return AddLimitsToDisplay(line, op, 0);
+          return AddLimitsToLargeOperatorDisplay(line, op, 0);
 
       }
     }
+    private IDisplay<TFont, TGlyph> MakeOperator(MathAtom op) {
+          // create a regular node.
+          var glyphs = _context.GlyphFinder.FindGlyphs(_font, op.Nucleus);
+          var glyphRun = new AttributedGlyphRun<TFont, TGlyph>(op.Nucleus, glyphs, _styleFont);
+          var run = new TextRunDisplay<TFont, TGlyph>(glyphRun, op.IndexRange, _context);
+          var runs = new List<TextRunDisplay<TFont, TGlyph>> { run };
+          var atoms = new List<IMathAtom> { op };
+          var line = new TextLineDisplay<TFont, TGlyph>(runs, atoms) {
+            Position = _currentPosition
+          };
+          return line;

```

Comment:
seems like there is missing adjustment for `_currentPosition`. I.e. something like that:

```cs
_currentPosition.X += line.Width;
if (op.Subscript != null || op.Superscript != null) {
  MakeScripts(op, line, op.IndexRange.Location, 0);
}
      
return line;
```

I would pulled out code that exactly the same for `Operator` and `LargeOperator` into a separate function. Then it will be easier to perceive.

Also, I'm unsure why separate case when op.Nucleus.Length is 1 for Operator is skipped.

Reply:
> I would pulled out code that exactly the same for Operator and LargeOperator into a separate function. Then it will be easier to perceive.

Done

> seems like there is missing adjustment for _currentPosition

Done. The failures happened when I removed MakeOperator and AddLimitsToOperatorDisplay because I thought there is nothing special about Operators and how they render subscripts and superscripts. When I removed them display was fine but the editor broke.